yarn.lock

```diff
@@ -3388,14 +3388,14 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@playwright/test@npm:^1.50.1":
-  version: 1.50.1
-  resolution: "@playwright/test@npm:1.50.1"
+"@playwright/test@npm:^1.52.0":
```

Comment:
Bumping the `@playwright/test` package requires a [commensurate bump to the Dockerfile](https://github.com/openmrs/openmrs-esm-patient-management#updating-playwright). Those two versions have to match.